.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: cpp
+
+sudo: required
+
+dist: focal
+
+matrix:
+  include:
+    - os: linux
+      dist: bionic
+      compiler: clang
+
+    - os: linux
+      dist: focal
+      compiler: clang
+
+before_script:
+    - sudo -E apt update -y
+    - sudo -E apt install -y libboost-dev
+
+script:
+    - . ./build.sh
```

Comment:
Why `. ./build.sh` not just `./build.sh`

Reply:
For unknown reasons `build.sh` is lack of shebang, so it is not really a shell script, it's just a set of commands in a file. Dot loads a source of a file into the current shell script and executes them. Dot has the same meaning as command `source` in Bash, but `.` is a POSIX standard while `source` is not, i.e., it's a Bashism.